shell.nix

```diff
@@ -0,0 +1,9 @@
+with import <nixpkgs> {};
+
+pkgs.mkShell {
+  buildInputs = [
+    libsass
+    openssl
+    pkgconfig
+  ];
+}
```

Comment:
Ah sorry I was thinking more of putting that in the documentation rather than in the repository

Reply:
ok, I'll put it as a snippet below the bullet point for nixOS.